### PR TITLE
feat: [CDS-44701]: freeze active warning instead of disable run button

### PR DESCRIPTION
**Pull request by udayvunnam**
### Summary

image

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by udayvunnam** [comment]
retrigger titlecheck

File: src/modules/70-pipeline/strings/strings.en.yaml [hunk 1]
```diff
@@ -1500,7 +1500,7 @@ customDeployment:
 step:
   validation:
     namePatternIsNotValid: 'Name can only contain alphanumerics, _ and -'
-runDisabledOnFreeze: Cannot run pipeline due to an active freeze
+runDisabledOnFreeze: An active freeze is applicable on this pipeline. Any deployment stage on this pipeline will be aborted.
```
Comment (Sagar-Rajput7) [inline src/modules/70-pipeline/strings/strings.en.yaml:1503]: Nit: not sure but this sentence conveys a different msg than what is intended here, 
"is applicable" -- can we have something like "is applied" or something else here, which denotes that freeze is already applied/enabled on this pipeline.

Ignore this if, I have misunderstood something here.

**Review by Sagar-Rajput7 — APPROVED** [review]

File: src/modules/70-pipeline/strings/strings.en.yaml (hunk 1 above)
Comment (udayvunnam) [reply src/modules/70-pipeline/strings/strings.en.yaml:1503]: This was a product call to add this text. Will share this suggestion.

**Comment by udayvunnam** [comment]
retrigger cypress-rest

**Comment by udayvunnam** [comment]
retrigger cypress-rest

**Review by rameshx — APPROVED** [review]

**Pull request merged**